src/App.js

```diff
@@ -0,0 +1,9 @@
+import Calculator from './components/Calculator';
+
+const App = () => (
+  <div>
+    <Calculator />
+  </div>
+);
```

Comment:
- [ ] Please convert App.js to a class based component

Reply:
Hi, @AshakaE. Thanks for your review. When I convert my component to a class-based component, a linter error appears saying that components should be written as a pure function. How do I deal with this problem?